content/en/integrations/faq/how-to-import-datadog-resources-into-terraform.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Importing Datadog Resources into Terraform
+kind: faq
+---
+
+## Overview
+
+Terraform supports an out-of-the-box way to import existing resources into your terraform state via the [`terraform import`][1] command.
+This can be done via the `terraform import <resource_type>.<resource_name> <existing_id>`
+
+This approach is `state only` and requires already having the HCL resource fully defined in your terraform configuration files. To import the configuration fully, you can use a tool like Terraformer
+
+## Terraformer
+
+The [terraformer project][2] allows you to import a resource as both state and HCL configuration.
+
+Once installed, you can setup a terraform directory with a basic `main.tf`
+
+This uses terraform 0.13+ syntax, but you can find more configurations on the [official datadog provider docs][3]
+
+```hcl
+# main.tf
+
+terraform {
+  required_providers {
+    datadog = {
+      source  = "DataDog/datadog"
+    }
+  }
+}
+
+# Configure the Datadog provider
+provider "datadog" {
+  api_key = var.datadog_api_key
+  app_key = var.datadog_app_key
+}
+
+variable "datadog_api_key" {
+  default = "<YOUR_API_KEY>"
+}
+
+variable "datadog_app_key" {
+  default = "<YOUR_APPLICATION_KEY>"
+}
+```
+
+Then run `terraform init` from within this directory to pull the datadog terraform provider.
+
+Now you can use `terraformer` to start importing resources. For example, to import Dashboard `abc-def-ghi` you can run
+
+`terraformer import datadog --resources=dashboard --filter=dashboard=abc-def-ghi --api-key <YOUR_API_KEY> --app-key <YOUR_APP_KEY>`
+
+This generates a folder `genrated` that contains both a terraform state file, as well as HCL terraform config files representing the imported resource.
```

Comment:
```suggestion
This generates a folder `generated` that contains both a terraform state file, as well as HCL terraform config files representing the imported resource.
```

Reply:
Maybe having an list of the files generated would be useful?

Reply:
Added a section about this. The `generated` folder structure can change depending on what flags you pass to `terraformer`, but I documented what you get if you run the commands in this FAQ